.travis.yml

```diff
@@ -36,7 +36,7 @@ env:
         - SETUP_CMD='test --coverage'
         - CONDA_CHANNELS='sunpy'
         - CONDA_DEPENDENCIES='openjpeg Cython jinja2 scipy matplotlib pytest==3.6 mock requests beautifulsoup4 sqlalchemy scikit-image pytest-mock lxml pyyaml pandas pytest-astropy suds-jurko glymur pytest-xdist dask drms sphinx-astropy pytest-cov hypothesis'
-        - PIP_DEPENDENCIES='pytest-rerunfailures sunpy-sphinx-theme pytest-sugar'
+        - PIP_DEPENDENCIES='pytest-rerunfailures sunpy-sphinx-theme pytest-sugar git+https://github.com/Cadair/parfive'
```

Comment:
You have it in pip already, shouldn't just depend from there?

Reply:
eventually, but at the moment I just need to keep it upto date with master.